scripts/capture.py

```diff
@@ -0,0 +1,31 @@
+import cv2
+import datetime
+import os
+
+OUTPUT_DIR = '/home/pi/data'
+DIR = '%Y-%m-%d'
+FILE = '%H:%M:%S.png'
+
+cap = cv2.VideoCapture(0)
+
+#  warmup camera
+for i in range(15):
```

Comment:
I would strongly recommend to add or configuration file or (and this is my preferred option) a way tto set it from cli maybe something like `--warmup 15` or `-w 15`